src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -5,8 +5,12 @@
     <IncludeSymbols>false</IncludeSymbols>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <PackageTags>aspnetcore</PackageTags>
-    <Description>Microsoft.AspNetCore.All</Description>
     <EnableApiCheck>false</EnableApiCheck>
+    <PackageDescription>Provides a default set of API for building an ASP.NET Core application, and also includes API for third-party integrations with ASP.NET Core.
+
+This package requires the ASP.NET Core runtime. This runtime is installed by the .NET Core SDK,
+or can be acquired separately using installers available at https://aka.ms/dotnet-download.
```

Comment:
Should there be an explicit newline splitting this sentence? Would that render weird on NuGet.org? I suggest having it be one line and let the UI stuff do the wrapping.